content/rs/release-notes/rs-6-4-2.md

```diff
@@ -12,6 +12,100 @@ aliases: [ /rs/release-notes/rs-6-4-2/,
+
+#### Allow port configuration for gossip_envoy service (RS72028) 
+
+You can use the REST API to configure the `gossip_envoy_admin_port` during bootstrapping or runtime. See the [cluster object reference]({{<relref "/rs/references/rest-api/objects/cluster">}}) for more details.
+
+Support for `rladmin` will be added in a future 6.4.2 maintenance release.
+
+### Known operating system limitations
+
+#### RHEL 7 and RHEL 8
+
+RS95344 - CRDB database will not start on Redis Enterprise v6.4.2 with a custom installation path.
+
+For a workaround, use the following commands to add the relevant CRDB files to the Redis library:
+
+```sh
+$ yum install -y chrpath
+$ find $installdir -name "crdt.so" | xargs -n1 -I {} /bin/bash -c 'chrpath -r ${libdir} {}'
+```
+
+This limitation was fixed in Redis Enterprise v6.4.2-43 (March maintenance release).
+
+#### Ubuntu 20.04
+
+- By default, you cannot use the SHA1 hash algorithm ([OpenSSL’s default security level is set to 2](https://manpages.ubuntu.com/manpages/focal/man3/SSL_CTX_set_security_level.3ssl.html#notes)). The operating system will reject SHA1 certificates even if the `mtls_allow_weak_hashing` option is enabled. You need to replace SHA1 certificates with newer certificates that use SHA-256. Note that the certificates provided with Redis Enterprise Software use SHA-256.  
+
+- RS95824 - Can't run `rl_rdbconvert` manually on Debian-based systems
+
+### Redis modules 
+
+Redis Enterprise Software v6.4.2 includes the new features delivered in the latest [Redis Stack release (v6.2.6)](https://redis.com/blog/introducing-redis-stack-6-2-6-and-7-0-6/ ):
+
+- [RediSearch v2.6.4](https://docs.redis.com/latest/modules/redisearch/release-notes/redisearch-2.6-release-notes/#v264-december-2022)
+
+- [RedisJSON v2.4.5](https://docs.redis.com/latest/modules/redisjson/release-notes/redisjson-2.4-release-notes/#v245-february-2023)
+
+- [RedisBloom v2.4.3](https://docs.redis.com/latest/modules/redisbloom/release-notes/redisbloom-2.4-release-notes/#v24-ga-v243-november-2022)
+
+- [RedisGraph v2.10.5](https://docs.redis.com/latest/modules/redisgraph/release-notes/redisgraph-2.10-release-notes/#v2105-december-2022)
+
+- [RedisTimeSeries v1.8.5](https://docs.redis.com/latest/modules/redistimeseries/release-notes/redistimeseries-1.8-release-notes/#v185-january-2023)
+
+### Resolved issues
+
+- RS95344 - Fix custom installation linkage bug for files related to Active-Active
+
+- RS91554 - Enhance visibility of `redis_mgr` 
+
+- RS90496 - Improve cross-node communication to support failovers
+
+- RS93004 - Allow positive `audit_port` configuration
+
+- RS58918 - Reduce the number of blocked clients by blocking only clients requested to flush until completion. Other clients won’t be affected.
```

Comment:
```suggestion
- RS58918 - Reduce the number of blocked clients by blocking only clients requested to flush until completion. Other clients won't be affected.
```

Reply:
@rrelledge This first sentence is a bit unclear to me. Does it make full sense to you?

Reply:
@nermiller I agree this was a bit confusing. I looked up the ticket associated with this fix for more context and tried to reword it:

"Reduce the number of clients blocked by flushing a database. Only the client that requested the flush will be blocked until the action finishes. Other clients won't be affected."

Reply:
Crystal clear now!